app/assets/javascripts/controllers/catalog/catalog_item_form_controller.js

```diff
@@ -247,6 +251,8 @@ ManageIQ.angular.app.controller('catalogItemFormController', ['$scope', 'catalog
         }
       }
     }
+    if (vm.catalogItemModel.provisioning_playbook_ttl !== undefined)
```

Comment:
@AparnaKarve it can be undefined when editing existing records, and when the value for the textbox is not provided